.github/workflows/api.yml

```diff
@@ -30,4 +30,31 @@ jobs:
       - name: Test massa API
         uses: matt-ball/newman-action@master
         with:
-          collection: api/test/massa_endpoints.postman_collection.json
\ No newline at end of file
+          collection: api/test/massa_endpoints.postman_collection.json
+
+  test-api-robot:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout project directory
+        uses: actions/checkout@v3
+      - name: Install RobotFramework tests dependencies
+        working-directory: api/test
+        run: |
+          pip install -r requirements.txt
+      - name: Install dependencies
+        uses: ./.github/actions/install
+        with:
+          os: ubuntu-latest
+      - name: Build thyra binary
+        run: GOOS=linux GOARCH=amd64 CGO_ENABLED=1 CC=gcc CXX=g++ go build cmd/thyra-server/main.go
```

Comment:
I don't think all theses options are needed `GOOS=linux GOARCH=amd64 CGO_ENABLED=1 CC=gcc CXX=g++`
I know CGO_ENABLED is 1 by defautl. for system, arch, and libc it will automatically take whats is available on current system.
Can you just try to remove, and see if its work?